packages/map/examples/test-single-county-file.json

```diff
@@ -0,0 +1,82 @@
+[
```

Comment:
could this go in storybook or should it be attached to the ticket? otherwise delete